player/lua/osc.lua

```diff
@@ -119,6 +119,35 @@ local user_opts = {
     -- luacheck: pop
 }
 
+local icon_font = "mpv-osd-symbols"
+
+-- Running this in Lua 5.3+ or LuaJIT converts a hexadecimal Unicode code point
+-- to the decimal value of every byte for Lua 5.1 and 5.2 compatibility:
+-- glyph='\u{e000}' output=''
+-- for i = 1, #glyph do output = output .. '\\' .. string.byte(glyph, i) end
+-- print(output)
+local icons = {
+    prev = "\238\132\144",           -- E110
+    next = "\238\132\129",           -- E101
+    pause = "\238\128\130",          -- E002
+    play = "\238\132\129",           -- E101
+    play_backward = "\238\132\144",  -- E110
```

Comment:
Can we align `=` vertically?

Reply:
I thought of aligning them but assignments in other tables in the file are not aligned, do you want to align only this one?

Reply:
it would look nice with constant value length, but whatever. avih also don't like aligning